<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Vijay" failures="1" tests="4" name="Test" time="0.084" errors="0" timestamp="2023-02-13T01:21:49 IST">
  <testcase classname="jan21TestNg.SkipTest" name="login" time="0.014"/>
  <testcase classname="jan21TestNg.SkipTest" name="logout" time="0.002"/>
  <testcase classname="jan21TestNg.SoftAssertDemo" name="test1" time="0.002"/>
  <testcase classname="jan21TestNg.SoftAssertDemo" name="test2" time="0.011">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at jan21TestNg.SoftAssertDemo.test2(SoftAssertDemo.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- Test -->
